scenes/menus/storybook/components/quest.gd

```diff
@@ -21,10 +21,10 @@ extends Resource
 
 @export_group("Advanced")
 
-## The path to the first scene of the quest. If you are following the structure of the template,
-## leave this as the default path. The path can be absolute (beginning with [code]res://[/code])
+## The path to the first scene of the quest.
+## The path can be absolute (beginning with [code]res://[/code])
 ## or relative to the folder where this quest resource is saved.
```

Comment:
It's no longer useful to have this be a relative path because learners will need to change it because they are renaming the intro scene… as a follow up commit later I will simplify this.